packages/app/src/messages/errors/mark-threads-read-status.error.ts

```diff
@@ -6,7 +6,7 @@
 import { MessagesEvent } from "App/messages/constants"
 
 export class MarkThreadsReadStatusError extends Error {
-  public type = MessagesEvent.ToggleThreadReadStatus
+  public type = MessagesEvent.ToggleThreadsReadStatus
```

Comment:
The type should be an `error type` not an event

Reply:
https://github.com/mudita/mudita-center/pull/753/commits/913da377623a7dede0fa02fac50762d2a117835a